blockly-ws-search/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "author": "Blockly Team",
+  "name": "@blockly/plugin-workspace-search",
+  "version": "1.20200304.0",
+  "description": "Implementation of workspace search for Blockly.",
+  "keywords": [
+    "blockly",
+    "workspace",
+    "search"
+  ],
+  "homepage": "https://github.com/google/blockly-samples/tree/workspace_search//blockly-ws-search#readme",
+  "bugs": {
+    "url": "https://github.com/google/blockly-samples/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/google/blockly-samples.git",
+    "directory": "blockly-ws-search"
+  },
+  "license": "Apache-2.0",
+  "main": "./dist/workspace-search.umd.js",
+  "unpkg": "./dist/workspace-search.umd.js",
+  "directories": {
+    "dist": "dist",
+    "src": "src"
+  },
+  "files": [
+    "dist",
+    "src"
```

Comment:
Do we still need to publish sources?

Reply:
Didn't you say we needed them for source maps?

Reply:
Right..